src/www/ui_components/add-server-view.js

```diff
@@ -298,15 +320,21 @@ Polymer({
     var input = event.target;
     input.toggleClass('input-invalid', input.invalid);
     if (input.invalid) {
-      this.$.addServerFooter.hidden = true;
-      this.$.invalidAccessKeyFooter.hidden = false;
+      this.invalidAccessKeyInput = true;
```

Comment:
*minor* `this.invalidAccessKey = input.invalid`

Reply:
done